grades/admin.py

```diff
@@ -34,17 +34,55 @@ class FinalGradeAuditAdmin(admin.ModelAdmin):
     def has_add_permission(self, *args, **kwargs):  # pylint: disable=unused-argument
         return False
 
+    def has_change_permission(self, *args, **kwargs):  # pylint: disable=unused-argument
```

Comment:
We left this out originally because it causes the admin to disappear entirely. Instead we set `readonly_fields` to be all of the fields